src/components/button/index.d.ts

```diff
@@ -33,6 +33,8 @@ declare module '@elastic/eui' {
     size?: ButtonSize;
     isLoading?: boolean;
     isDisabled?: boolean;
+    contentProps?: object;
```

Comment:
use `HTMLAttributes<HTMLSpanElement>` for this & the other three to enforce the values are valid attributes of spans.
```suggestion
    contentProps?: HTMLAttributes<HTMLSpanElement>;
```

Reply:
Ahhh ok, thx!